### ISSUE TITLE
Support Pad "circular" Mode

### ISSUE THREAD

**Issue by leimao** [onnx/onnx#4768]
PyTorch supports Pad "circular" mode. I wonder if it's possible to add this to the Pad ONNX operator?
https://pytorch.org/docs/stable/generated/torch.nn.functional.pad.html#torch.nn.functional.pad

**Comment by emojilearning** [comment]
I think it makes sense. For panorama images, (horizontal) circular padding is commonly used in a variety of application.

**Comment by leimao** [comment]
Circular padding has been supported via the `wrap` mode.
